packages/lab/stories/form-field-legacy/form-field.doc.stories.mdx

```diff
@@ -16,8 +17,10 @@ import {
 # Form Field Legacy
 
 <Banner status="warning">
-  This component has been deprecated. The new [Form
-  Field](?path=/docs/documentation-lab-form-field-next--page) is a current WIP.
+  <BannerContent>
+    This component has been deprecated. The new [Form
+    Field](?path=/docs/documentation-lab-form-field--page) is a current WIP.
```

Comment:
```suggestion
    Field](?path=/docs/documentation-core-form-field--page) is a current WIP.
```

I guess you checked the live storybook, but it is actually in core now if you check the sb in this PR (I don't understand when it updates on the 'live' version?)